addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/SourceCodeDisclosureScanRuleUnitTest.java

```diff
@@ -107,6 +109,41 @@ void givenSQLAndPhpCodeThenOnlyOneAlertRaised() {
         assertThat(alertsRaised.size(), is(1));
     }
 
+    @Test
+    void shouldNotRaiseAlertOnCssRequest() throws Exception {
+        // Given
+        msg.getRequestHeader().setURI(new URI("https://www.example.com/assets/styles", true));
+        msg.getRequestHeader().setHeader(HttpResponseHeader.CONTENT_TYPE, "text/css");
```

Comment:
`HttpResponseHeader` → `HttpHeader` (can also be replaced in the other tests), it's clearer if accessed with the non-specific header.

Reply:
@thc202 Modified code to use HttpHeader. Please review latest changes. Hopefully now it is in acceptable state. Thank you for your guidance and help!